src/Test.UnitTests.Sarif/HashUtilitiesTests.cs

```diff
@@ -72,7 +72,7 @@ public void HashUtilities_ShouldRespectQuietFlag()
                 //  Do nothing, this is the desired behavior.
             }
 
-            //  Mimic the --quiet flag.  We should not use the console this time.
+            //  Mimic the '--quiet true'.  We should not use the console this time.
```

Comment:
>quiet [](http://example.com/codeflow?start=29&length=5)

Again all of these values s/b legal:

--quiet (s/be interpreted as --quiet true)
-- quiet true
--quiet false